src/PublicApiAnalyzers/UnitTests/DeclarePublicApiAnalyzerTests.cs

```diff
@@ -2167,7 +2206,7 @@ public class {|RS0016:{|RS0016:C2|}|} { }
             await VerifyCSharpAdditionalFileFixAsync(source, shippedText, unshippedText, fixedUnshippedText);
         }
 
-        [WindowsOnlyTheory]
+        [Theory]
```

Comment:
I expect this test to be fixed on Linux now. Waiting for CI Linux build to confirm.